templates/systemd_dropin_rhel8.erb

```diff
@@ -0,0 +1,4 @@
+[Service]
+ExecStartPre=
+ExecStartPre=-/bin/chown -R radiusd.radiusd /var/run/radiusd
+ExecStartPre=/usr/sbin/radiusd -C
```

Comment:
As a style thing, can we add a newline at the end of this file and potentially update the filename